go.mod

```diff
@@ -0,0 +1,33 @@
+module github.com/russellcardullo/terraform-provider-pingdom
+
+go 1.12
+
+require (
+	github.com/apparentlymart/go-cidr v1.0.0 // indirect
+	github.com/armon/go-radix v1.0.0 // indirect
+	github.com/aws/aws-sdk-go v1.16.22 // indirect
+	github.com/blang/semver v3.5.1+incompatible // indirect
+	github.com/hashicorp/go-getter v1.0.0 // indirect
+	github.com/hashicorp/go-hclog v0.0.0-20190109152822-4783caec6f2e // indirect
+	github.com/hashicorp/go-plugin v0.0.0-20181212150838-f444068e8f5a // indirect
+	github.com/hashicorp/go-uuid v1.0.1 // indirect
+	github.com/hashicorp/go-version v1.1.0 // indirect
+	github.com/hashicorp/hcl v1.0.0 // indirect
+	github.com/hashicorp/hcl2 v0.0.0-20190116200548-7b147fbae47a // indirect
+	github.com/hashicorp/hil v0.0.0-20170627220502-fa9f258a9250 // indirect
+	github.com/hashicorp/terraform v0.11.11
```

Comment:
I had to update to terraform 0.12.3 here to fix some issues with newer terraform.  Could you pull in the go.mod from the master branch and do a `go mod tidy`?